src/source.c

```diff
@@ -1532,8 +1532,13 @@ struct dispatch_timer_s _dispatch_timer[] =  {
 #define DISPATCH_TIMER_COUNT \
 		((sizeof(_dispatch_timer) / sizeof(_dispatch_timer[0])))
 
+#if __linux__
```

Comment:
I don't understand that guard, why isn't uintptr_t working on linux, why is it needing a pointer type?